### ISSUE TITLE
Share link function for organization pages

### ISSUE THREAD

**Issue by Solisol** [hackforrefugees/bulk-giving#7]
Add a nice share-to-social-media thingy where users can share an organization page (with all their needs) to a social media of their choice, or get a human readable link.

This would include setting up some simple default metadata so that pages like facebook will display something uselful.

**Comment by Unarmed** [comment]
Links exist, but are somewhat broken. The router unfortunatley routes prior to meteor having loaded it's stuff. It's a bug I've been meaning to look into, maybe we can get aronud it using a timeout or something.

**Comment by Unarmed** [comment]
I guess it would also be nice if an organization could choose their URL as the one they are assigned can be a bit wonky.

**Comment by Solisol** [comment]
Yes, those are good ideas to look into, but the point of this issue would be a nice button "Share link to this organization" that both givers and organizations can use to post to facebook, twitter etc.

**Comment by Unarmed** [comment]
Gotcha.
